instrumentation/thrift/thrift-common/library/src/main/java/io/opentelemetry/instrumentation/thrift/common/ThriftSpanNameExtractor.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.thrift.common;
+
+import io.opentelemetry.instrumentation.api.instrumenter.SpanNameExtractor;
+
+public final class ThriftSpanNameExtractor implements SpanNameExtractor<ThriftRequest> {
```

Comment:
there are semantic conventions that define the name for a RPC span, you should follow them. See `RpcSpanNameExtractor`.